modules/broker-common/src/main/java/io/ballerina/messaging/broker/common/EventConstants.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package io.ballerina.messaging.broker.common;
+
+/**
+ * Contains commonly used EventNames.
+ */
+public final class EventConstants {
+
+    public static final int CONSUMER_ADDED_EVENT = 1;
+    public static final int MESSAGE_PUBLISHED_EVENT = 2;
+    public static final int QUEUE_CREATED = 3;
+    public static final int BINDING_CREATED = 4;
```

Comment:
It is better not to have these constants in the common module since they are specific to the broker core.